indent/verilog_systemverilog.vim

```diff
@@ -169,12 +173,9 @@ function GetVerilog_SystemVerilogIndent()
   " or if it starts with a `define call, which might not require the ; end
   elseif (
     \ last_line =~ ';' . vlog_comment . '*$' ||
-    \ last_line =~ '^\s*`\k\+'
-    \ ) &&
```

Comment:
I agree with this change, but I would prefer not to touch the original code, such that I can run `git blame` to see when it was added.